touch.js

```diff
@@ -0,0 +1,8 @@
+let checkDevice = navigator.userAgent;
+let touch = document.querySelector('#keyboard-controls-help');
```

Comment:
I think it's better to name this variable `keyboardControlsHelp` :)

It's not touch, it's help for keyboard controls.

So I don't think we should call it "touch".